scarlet/component.py

```diff
@@ -87,8 +87,8 @@ def __init__(self, sed, morph, prior=None, fix_sed=False, fix_morph=False):
         # set sed and morph
         self._sed = np.array(sed)
```

Comment:
I think we can expose sed and morph again since they're are ordinary numpy arrays now

Reply:
I like hiding them so that users don't try to modify them in place and get unexpected results.